stripe/res/layout/card_input_view.xml

```diff
@@ -1,159 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools"
-              android:orientation="horizontal"
-              android:layout_width="match_parent"
-              android:layout_height="wrap_content"
-              >
-
-    <ImageView
-        android:id="@+id/iv_card_icon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/card_icon_padding"
-        android:layout_marginBottom="@dimen/card_icon_padding"
-        android:layout_marginEnd="@dimen/card_icon_padding"
-        android:layout_marginRight="@dimen/card_icon_padding"
-        android:src="@drawable/stp_card_placeholder_template"
-        />
-
-    <RelativeLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_vertical"
-        >
-
-        <com.stripe.android.view.LockableHorizontalScrollView
-            android:id="@+id/root_scroll_view"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center_vertical"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentStart="true"
-            android:fillViewport="true"
-            android:fadeScrollbars="false"
-            tools:importantForAccessibility="no"
-            android:scrollbars="none">
-
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="horizontal"
-                tools:importantForAccessibility="no"
-                >
-
-                <FrameLayout
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    tools:importantForAccessibility="no"
-                    >
-
-                    <!--The only purpose of this TextView is to be an accessibility label.-->
-                    <TextView
```

Comment:
This hierarchy goes six layers deep (and secretly a seventh)! **We can do better.**